passwords/genPass.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+
+function check_prereq(){
+    if ! command -v docker --version &> /dev/null
+    then
+        error_exit "docker could not be found."
+    elif [ "$(docker ps | grep -c 'CONTAINER ID')" == 0 ]
+    then
+        error_exit "docker is not running."
+    fi
+}
+
+function gen_password(){
+    env LC_ALL=C tr -dc 'A-Za-z0-9_' < /dev/urandom | head -c "$1"
+}
+
+function sign_enc_keys(){
+    sign=$(sudo docker run circleci/server-keysets:latest generate signing -a stdout)
+    enc=$(sudo docker run circleci/server-keysets:latest generate encryption -a stdout)
```

Comment:
Maybe a nit: We only need sudo in one place - here or the execution of the script

Reply:
Actually the more I think about it the more we should remove this from the script. If the user needs sudo they can run the entire script as sudo.

Reply:
removed the sudo from pipe ( in readme)